app/services/prison_visits/client.rb

```diff
@@ -43,7 +43,7 @@ def request(method, route, params)
 
       response = @connection.request(options)
 
-      JSON.parse(response.body)
+      response.body.present? ? JSON.parse(response.body) : nil
```

Comment:
Ok, I'll update it, although it will be strange to be request bodies into JSON to then discarded